code/modules/supply/supply_packs/pack_security.dm

```diff
@@ -298,6 +298,14 @@
 	containertype = /obj/structure/closet/crate/secure/plasma
 	containername = "disabler smg crate"
 
+/datum/supply_packs/security/armory/securiblades
+	name = "Securiblade Crate"
+	contains = list(/obj/item/storage/belt/sheath/secsword,
+		/obj/item/storage/belt/sheath/secsword)
+	cost = 550
+	containertype = /obj/structure/closet/crate/secure/plasma
```

Comment:
why not a security crate?

Reply:
Just about all security weapons, minus ammo and WTs, come in the `/obj/structure/closet/crate/secure/plasma` variant of the crates. Since this blade is energy-themed, I figured it should also be with that type.

Reply:
makes sense